### PR TITLE
Fix generated Pulumi.yaml on convert

### PR DESCRIPTION
**Pull request by aq17**
Fixes `pu/pu`#[10795](https://github.com/pulumi/pulumi/issues/10795)

### CODE REVIEW

**Review by Frassle — COMMENTED** [review]
Change looks ok, is there a framework in pulumi-yaml for testing eject by itself yet?

**Comment by aq17** [comment]
> Change looks ok, is there a framework in pulumi-yaml for testing eject by itself yet?

Not standalone for eject, but could possibly add a check [here](https://github.com/pulumi/pulumi-yaml/blob/main/pkg/tests/example_transpile_test.go#L283-L285)?

File: pkg/pulumiyaml/codegen/eject.go [hunk 1]
```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
 )
 
+var ProjectKeysToOmit = []string{"configuration", "resources", "outputs"}
```
Comment (AaronFriel) [inline pkg/pulumiyaml/codegen/eject.go:20]: I think `variables` would also be excluded here?

**Review by AaronFriel — COMMENTED** [review]
I believe there's another key to omit, otherwise a fairly clear change here.

**Review by AaronFriel — APPROVED** [review]
bors merge

**Review by AaronFriel — APPROVED** [review]
LGTM!

**Pull request merged**